packages/mdc-chips/chip/mdc-chip.scss

```diff
@@ -22,17 +22,29 @@
 .mdc-chip {
   @include mdc-ripple-surface;
   @include mdc-ripple-radius-bounded;
+  @include mdc-states;
+  @include mdc-states-activated(black);
+  
   @include mdc-chip-corner-radius($mdc-chip-border-radius-default);
   @include mdc-chip-fill-color($mdc-chip-fill-color-default);
   @include mdc-chip-ink-color($mdc-chip-ink-color-default);
 
   display: inline-block;
+  border-radius: 16px;
+  background: rgba(0, 0, 0, 0.08);
```

Comment:
Can you use the theme mixins for these properties?

Reply:
Whoops, bad merge. Done!